signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1049,6 +1052,102 @@ async fn run_dkg_from_scratch() {
     }
 }
 
+/// Tests that dkg will not be triggered if signer set didn't change
+#[test(tokio::test)]
+async fn dont_run_dkg_if_signer_set_didnt_change() {
```

Comment:
Both of these tests need to drop the database at the end.

Reply:
[4a3a399](https://github.com/stacks-sbtc/sbtc/pull/1602/commits/4a3a399fbafeee758c35b2812f2d2c94d4459ac8)